velox/docs/configs.rst

```diff
@@ -164,4 +167,35 @@ data size exceeds this limit, otherwise it spills the partition with most data.
 If the limit is zero, then the spiller always spills a previously spilled
 partition if it has any data. This is to avoid spill from a partition with a
 small amount of data which might result in generating too many small spilled
-files.
\ No newline at end of file
+files.
+
+
+Hive Configuration Properties
+-----------------------------
+
+Table Writer
+~~~~~~~~~~~~
+
+``max_partitions_per_writers``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``100``
+
+Maximum number of partitions per writer.
+
+``immutable_partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``true``
+
+Can new data be inserted into existing partitions or existing unpartitioned tables.
```

Comment:
The property name "immutable_partitions" suggests that it applies only to partitioned tables, but the description suggests it applies to unpartitioned tables as well. Is this correct?

Reply:
Yes, this immutable_partitions config also have effect on the path for unpartitioned table. Cross-reading the presto java oss code I got the impression that the unpartitioned table might be treated as single partitioned table.